ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -19,10 +19,15 @@ import U from '../../../Core/Utilities.js';
 const {
     extend,
     isNumber,
-    merge
+    merge,
+    pick
 } = U;
 
 /* eslint-disable no-invalid-this, valid-jsdoc */
+interface ExtendedPositionObject extends PositionObject{
```

Comment:
@bre1470 could you please confirm if that is the optimal way of extending that interface?

Reply:
isn't it exactly this?
```js
{
x: number;
y: number;
xAxis?:number;
yAxis?:number;
}
```

Reply:
`x/yAxis?: number|string;` should be correct too. In theory `xAxis: chart.xAxis[index]` is also supported, but I have never tested it

I'm not sure if `ExtendedPositionObject` name is correct here, sorry for the confusion. Just checked, we already have `MockPointOptions` interface defined, let's reuse it